docs/source/user_guide/using_datasets.rst

```diff
@@ -2110,6 +2120,187 @@ some workflows when it is available.
 Dates and datetimes
 ___________________
 
+.. _builtin-datetime-fields:
+
+Builtin datetime fields
+-----------------------
+
+Datasets and samples have various builtin datetime fields that are
+automatically updated when certain events occur.
+
+The
+:attr:`Dataset.last_loaded_at <fiftyone.core.dataset.Dataset.last_loaded_at>`
+property tracks the datetime that the dataset was last loaded:
+
+.. code-block:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.zoo as foz
+
+    dataset = foz.load_zoo_dataset("quickstart")
+
+    print(dataset.last_loaded_at)
+    # 2025-05-04 21:00:45.559520
+
+The
+:attr:`Dataset.last_modified_at <fiftyone.core.dataset.Dataset.last_modified_at>`
+property tracks the datetime that dataset-level metadata was last modified,
+including:
+
+-   when properties such as
+    :attr:`name <fiftyone.core.dataset.Dataset.name>`,
+    :attr:`persistent <fiftyone.core.dataset.Dataset.persistent>`,
+    :attr:`tags <fiftyone.core.dataset.Dataset.tags>`,
+    :attr:`description <fiftyone.core.dataset.Dataset.description>`,
+    :attr:`info <fiftyone.core.dataset.Dataset.info>`, and
+    :attr:`app_config <fiftyone.core.dataset.Dataset.app_config>` are edited
+-   when fields are added or deleted from the dataset's schema
+-   when group slices are added or deleted from the dataset's schema
+-   when saved views or workspaces are added, edited, or deleted
```

Comment:
does it also include runs?

Reply:
good call, I initially intended to provide an incomplete list but at this point its close enough to being a full list I should list more things 🙃

Reply:
updated 👍